data-prepper-plugins/kafka-plugins/src/main/java/org/opensearch/dataprepper/plugins/kafka/producer/KafkaCustomProducer.java

```diff
@@ -87,7 +87,9 @@ public KafkaCustomProducer(final KafkaProducer producer,
         this.bufferedEventHandles = new LinkedList<>();
         this.expressionEvaluator = expressionEvaluator;
         this.tagTargetKey = tagTargetKey;
+        System.out.println("______________"+kafkaProducerConfig.getTopic());
```

Comment:
Please remove these `println` statements. Or change them to `LOG.debug`. There are a few of them in the PR.

Reply:
Sorry. My mistake. Removed them